.pip_readme.rst

```diff
@@ -27,6 +27,10 @@ for adjoint transformations where needed, and comes with different
 optimisations (precompute or not) that one may select depending on
 available resources and desired angular resolution $L$.
 
+As of version 1.0.2 `S2FFT` also provides PyTorch implementations of underlying 
+precompute transforms. In future releases this support will be extended to our 
+recursion based algorithms.
```

Comment:
The terminology we use in the paper in on-the-fly algorithms.  (Since the pre-compute is essentially also recursion based, we just precompute the recursion.)

Reply:
updated.